group_vars/all.yml

```diff
@@ -8,3 +8,5 @@ cifmw_project_dir_absolute: "{{ ansible_user_dir }}/{{ cifmw_project_dir }}"
 cifmw_installyamls_repos_relative: src/github.com/openstack-k8s-operators/install_yamls
 # since cifmw_installyamls_repos var already exists, let's use that and move all definition here in single place instead of creating another variable.
 cifmw_installyamls_repos: "{{ ansible_user_dir }}/{{ cifmw_installyamls_repos_relative }}"
+cifmw_architecture_repo_absolute: src/github.com/openstack-k8s-operators/architecture
```

Comment:
Seems odd to call this `absolute` when the same pattern is used for `install_yamls` on L8 and it is called `relative` there